test/js/ez-platform-ui-app.js

```diff
@@ -150,6 +150,78 @@ describe('ez-platform-ui-app', function() {
         });
     });
 
+    describe('handle contentDiscover', function () {
+        const property = 'anything';
+        let eventFired = false;
+
+        function fireContentDiscover () {
+            element.dispatchEvent(new CustomEvent('ez:contentDiscover', {
+                detail: {
+                    config: {
+                        anyProp: property,
+                    },
+                    handlers: {
```

Comment:
I agree with the change but  shouldn't this  be done with the implementation of <ez-browse> then ?